docs/content/users/quickstart.md

```diff
@@ -159,14 +159,17 @@ ddev launch
 
 ## Drupal
 
-=== "Drupal 10"
+=== "Drupal"
```

Comment:
not listed, but mentioned in the first sentence, drupal 8 is already failing on the `composer require` step due to the fact that php 8.2 is too new. (php 8.2 and mariadb 10.11 are set for drupal 8 with the drupal project type as well) 

and based on the results from this comment and the two in regards of drupal 9 and 11 the first sentence creates false expectations. if i read the first line i expect i choose `drupal` as the project type and ddev adjusts everything according to the project at hand (which is impossible since composer create is run at a later step)